src/main/java/com/google/neighborgood/helper/TaskGroup.java

```diff
@@ -19,101 +19,65 @@
 import com.google.appengine.api.datastore.Entity;
 import com.google.appengine.api.datastore.EntityNotFoundException;
 import com.google.appengine.api.datastore.Key;
-import com.google.appengine.api.datastore.KeyFactory;
 import com.google.appengine.api.users.UserService;
 import com.google.appengine.api.users.UserServiceFactory;
-import java.sql.Timestamp;
-import java.text.SimpleDateFormat;
+import com.google.neighborgood.User;
+import com.google.neighborgood.task.Task;
+import java.util.ArrayList;
 import java.util.HashMap;
 
 /**
  * Helper class that stores the HTML depiction of tasks in groups of 10 or less along with whether
  * or not the end of the query has been reached
  */
 public class TaskGroup {
-  private static HashMap<String, String>
-      usersNicknames; // Stores task owner's user info to prevent querying multiple times in
+  private static HashMap<String, User>
+      usersInfo; // Stores task owner's user info to prevent querying multiple times in
   // datastore for the same user's info
   private static DatastoreService datastore;
   private final boolean userLoggedIn;
-  private final String userId;
   private int currentTaskCount;
-  private StringBuilder tasks; // String-like representation of 10 tasks
   private boolean endOfQuery;
+  private ArrayList<Task> tasks;
```

Comment:
Same suggestion here. I would prefer `List<Task> tasks` to `ArrayList<Task> tasks`

Reply:
Will update this in the next commit, thanks!